test/models/user_test.rb

```diff
@@ -0,0 +1,43 @@
+require 'test_helper'
+
+class UserTest < ActiveSupport::TestCase
+  def setup
+    @user = User.new(name: "Example User", email: "user@example.com", password: "foobar", password_confirmation: "foobar")
+  end
+
+  test "name should not be too long" do
+    @user.name = "a" * 51
+    assert_not @user.valid?
+  end
+  test "email should not be too long" do
```

Comment:
```suggestion
  end
  
  test "email should not be too long" do
```
add a new line between your test cases